_faq/application_template.md

```diff
@@ -77,7 +77,7 @@ Other Commitments
 -----------------
 
 - Do you have any other commitments during the GSoC period,
-  May 8th to August 29th?
+  May 14th to August 6th?
```

Comment:
The end date is August 14th according to [gsoc website](https://developers.google.com/open-source/gsoc/timeline). Student should also be available to submit his report. Drop the colon after Fixes, visit [this](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) for more info.